CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## [3.1.1] - TBD
```

Comment:
Since this release will include a `feat`, the next version should be `3.2.0`

Reply:
You're right! Done.